drivers/sensor/amd_sb_tsi/sb_tsi.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2023 Google LLC
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT amd_sb_tsi
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/logging/log.h>
+#include "sb_tsi.h"
+
+LOG_MODULE_REGISTER(AMD_SB_TSI, CONFIG_SENSOR_LOG_LEVEL);
+
+static int sb_tsi_sample_fetch(const struct device *dev,
+			       enum sensor_channel chan)
+{
+	struct sb_tsi_data *drv_data = dev->data;
+	const struct sb_tsi_config *cfg = dev->config;
```

Comment:
Convention is to call these local variables `data` and `config` respectively. Please fix throughout

Reply:
Done